### PR TITLE
fix Fauxton docs

### PR DESCRIPTION
**Pull request by michellephung**
Please check since i've never made a PR to couchdb-docs before

### CODE REVIEW

File: src/fauxton/install.rst [hunk 1]
```diff
@@ -12,66 +12,44 @@
...
+.. _node.js: http://nodejs.org/
+.. _npm: https://npmjs.org/doc/README.html
 
-    ./bin/grunt couchapp_deploy
+Using the dev server is the easiest way to use Fauxton, specially when developing for it::
 
-The Fauxton be available by `/fauxton/_design/fauxton/index.html
-<http://localhost:5984/fauxton/_design/fauxton/index.html>`_
+    $ npm install -g fauxton
+    $ npm run dev
```
Comment (garrensmith) [inline src/fauxton/install.rst]: This only works in the couchdb-fauxton repo

File: src/fauxton/install.rst [hunk 2]
```diff
@@ -12,66 +12,44 @@
...
-Deploy Fauxton to your local CouchDB instance:
+.. _node.js: http://nodejs.org/
+.. _npm: https://npmjs.org/doc/README.html
 
-    ./bin/grunt couchapp_deploy
+Using the dev server is the easiest way to use Fauxton, specially when developing for it::
 
-The Fauxton be available by `/fauxton/_design/fauxton/index.html
-<http://localhost:5984/fauxton/_design/fauxton/index.html>`_
+    $ npm install -g fauxton
```
Comment (garrensmith) [inline src/fauxton/install.rst]: Do you mean `git clone fauxton-repo`

File: src/fauxton/install.rst [hunk 3]
```diff
@@ -96,6 +74,11 @@ layout, data, breadcrumbs and api point is required for the view.
 .. _backbone.js: http://backbonejs.org/
 .. _Backbone.layoutmanager: https://github.com/tbranyen/backbone.layoutmanager
 
+Writing Add-Ons
+---------------
+We have Fauxton Add-On documentation:
+    https://github.com/apache/couchdb-fauxton/blob/master/writing_addons.md
```
Comment (garrensmith) [inline src/fauxton/install.rst]: These addon docs are so old. We need to update them before linking to them. Maybe we should leave this out

**Comment by michellephung** [comment]
i think i fixed the issues can you check? i put them on the second commit

File: src/fauxton/install.rst [hunk 4]
```diff
@@ -12,66 +12,44 @@
...
+.. _node.js: http://nodejs.org/
+.. _npm: https://npmjs.org/doc/README.html
 
-    ./bin/grunt couchapp_deploy
+Using the dev server is the easiest way to use Fauxton, specially when developing for it::
 
-The Fauxton be available by `/fauxton/_design/fauxton/index.html
-<http://localhost:5984/fauxton/_design/fauxton/index.html>`_
+    $ git clone https://github.com/apache/couchdb-fauxton.git
+    $ npm run dev
```
Comment (garrensmith) [inline src/fauxton/install.rst]: I think they would need to do an `npm install` as well. so `npm install && npm run dev`

**Comment by garrensmith** [comment]
+1 after the small change

**Pull request merged**